add_sourceapp.sh

```diff
@@ -188,7 +194,10 @@ addlib() {
 checkpermissions() {
   apk="$1"
   echo "Checking if any extra privapp-permissions are needed..."
-  python scripts/verify-permissions.py "$apk"
+  missingperms=$(python3 scripts/verify-permissions.py "$apk" 2>&1 >/dev/null)
```

Comment:
does the python script give a return code? That is probably easy to check for than testing if it returns a string.